app/pages/project/stats/charts.cjsx

```diff
@@ -124,7 +124,17 @@ Graph = React.createClass
         if data.index % numberBars
           data.element.attr({style: "display: none"})
     else if data.type == 'bar'
-      data.element.attr({style: "stroke-width: #{100 / length}%"})
+      data.element.attr(
+        style: "stroke-width: #{100 / length}%"
+        focusable: true
```

Comment:
I actually don't know, just found it during a google search, removing it does not seem to hurt anything in Chrome.  Also I just noticed focus does not work at all in Firefox.